### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,23 @@ These instructions will get you a copy of the project up and running on your loc
...
+* [CentOS 7](https://wiki.centos.org) - Enterprise Linux
+* [ElasticSearch 6.8+](https://www.elastic.co) - High-performance Indexing system
 * [FOSElasticaBundle](https://github.com/FriendsOfSymfony/FOSElasticaBundle) - PHP integration for Symfony with ElasticSearch
-* [RabbitMQ](https://www.rabbitmq.com/documentation.html) - Open source message broker software
+* [PHP 7.1](http://php.net) - Scripting language, installed from Remi
+* [PostGIS 2.4](http://www.postgis.org) - Geospatial extensions for PostgreSQL
+* [PostgreSQL 9.6](https://www.postgresql.org) - ORDBMS
+* [RabbitMQ](https://www.rabbitmq.com) - Open source message broker
+* [Remi](https://rpms.remirepo.net) - RPM Repository to support current PHP on CentOS 7.
+* [Symfony 3.4](https://symfony.com) - PHP framework for web application
```
Comment (mickel1138) [inline README.md]: This is NOT a prerequisite. This just gets installed when you run composer, not before.

File: README.md [hunk 2]
```diff
@@ -9,20 +9,23 @@ These instructions will get you a copy of the project up and running on your loc
...
+* [Apache 2.4](https://www.apache.org) - Webserver
+* [CentOS 7](https://wiki.centos.org) - Enterprise Linux
+* [ElasticSearch 6.8+](https://www.elastic.co) - High-performance Indexing system
 * [FOSElasticaBundle](https://github.com/FriendsOfSymfony/FOSElasticaBundle) - PHP integration for Symfony with ElasticSearch
-* [RabbitMQ](https://www.rabbitmq.com/documentation.html) - Open source message broker software
+* [PHP 7.1](http://php.net) - Scripting language, installed from Remi
+* [PostGIS 2.4](http://www.postgis.org) - Geospatial extensions for PostgreSQL
+* [PostgreSQL 9.6](https://www.postgresql.org) - ORDBMS
+* [RabbitMQ](https://www.rabbitmq.com) - Open source message broker
+* [Remi](https://rpms.remirepo.net) - RPM Repository to support current PHP on CentOS 7.
```
Comment (mickel1138) [inline README.md]: I feel this is also not needed, we don't really want dictate how to install the needed software.

File: README.md [hunk 3]
```diff
@@ -9,20 +9,23 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-* [CENTOS 6/7](https://wiki.centos.org/)
-* [PHP 7.1](http://php.net/docs.php )
-* [Symfony 3.4](https://symfony.com/doc/3.4/index.html) - PHP framework for web application
-* [PostgreSQL 9.6](https://www.postgresql.org/docs/9.6/static/release-9-6.html) - ORDBMS
+* [Apache 2.4](https://www.apache.org) - Webserver
```
Comment (mickel1138) [inline README.md]: Not a big fan of a alphabetized listing, it should be be significance or install order rather.

File: README.md (hunk 1 above)
Comment (fightingtexasaggie) [reply README.md]: Good point, but Composer certainly is.

File: README.md (hunk 3 above)
Comment (fightingtexasaggie) [reply README.md]: re-ordering

**Comment by mickel1138** [comment]
Also know that there are significant changes to this readme file in drupal divorce.

**Comment by fightingtexasaggie** [comment]
not needed, at least not right now.

**Pull request closed without merging**